### DISCUSSION TITLE
RegEx Capture?

### DISCUSSION THREAD

**Discussion by HaydnChambers** [DraqueT/PolyGlot#1384] [Ideas]
I'm looking for a way to reduplicate a part of a word for adverbs in my language to imply a more emphasized meaning. I wish I could capture a part of the word and then have it go to the output, so "so" would become "soso" after capturing the whole word in this case, if that makes any sense. 
Maybe a new feature for automatic reduplication isn't needed, but I don't know how to do it otherwise. Does Polyglot even make use of captures in RegEx or is it just to change phonemes in simulated evolution and prefixes and infixes and stuff?

**Comment by hyacynthesizer** [comment]
This regex feature is fully implemented - check the Example Languages under the Help tab, specifically the "Setup Conjugation Infix Examples" option. Each capturing group in the regex test pattern can be recalled with $1, $2, etc. in the replacement pattern.
